cas-server-support-jpa-ticket-registry/src/main/java/org/jasig/cas/ticket/registry/JpaTicketRegistry.java

```diff
@@ -195,8 +200,19 @@ private Ticket getRawTicket(final String ticketId) {
         return tickets;
     }
 
+    /**
+     * Set the id prefix used by {@link TicketGrantingTicket} implementations.
+     *
+     * @param ticketGrantingTicketPrefix the prefix to use
+     * @deprecated As of 4.2. use {@link JpaTicketRegistry#setTicketGrantingTicketPrefixes} instead.
+     */
+    @Deprecated
     public void setTicketGrantingTicketPrefix(final String ticketGrantingTicketPrefix) {
```

Comment:
I'm fine with dropping them, I have no need for this in my deployment currently.

I have some custom OAuth code in my deployment (different from the stock OAuth support) that provides long term sessions that can bridge into the CAS protocol and be used to issue PTs. Right now it's standalone, but I've been thinking about reworking it into the `RegisteredService`/`Service`/`TicketGrantingTicket` concepts. I doubt I would have need to define another unique prefix for TGTs, if it does arise I'm fine with submitting a PR to re-add the methods at that time.